src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -156,15 +156,17 @@ def _describe(self, catalog: str, database: str, table: str) -> Table | None:
             for key, value, _ in self._fetch(f"DESCRIBE TABLE EXTENDED {full_name}"):
                 describe[key] = value
             return Table(
-                catalog=describe["Catalog"],
-                database=database,
-                name=table,
-                object_type=describe["Type"],
-                table_format=describe.get("Provider", "").upper(),
-                location=describe.get("Location", None),
-                view_text=describe.get("View Text", None),
-                upgraded_to=self._parse_table_props(describe.get("Table Properties", "")).get("upgraded_to", None),
-                storage_properties=self._parse_table_props(describe.get("Storage Properties", "")),
+                catalog=None if describe["Catalog"] is None else describe["Catalog"].lower(),
+                database=None if database is None else database.lower(),
+                name=None if table is None else table.lower(),
+                object_type=None if describe["Type"] is None else describe["Type"].lower(),
```

Comment:
```suggestion
                object_type=None if describe["Type"] is None else describe["Type"].upper(),
```

Constants are mostly upper